workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentLauncher.cs

```diff
@@ -270,10 +270,13 @@ private async Task<bool> TriggerUploadAssemblyAsync()
                     "upload",
                     uploadAssemblyName,
                     "--project_name",
-                    projectName
+                    projectName,
+                    "--json_output"
```

Comment:
This isn't directly related to PR, but cleans up output from `spatial` format (`time=... msg=...`)  to clear message